cmd/jnet/main.go

```diff
@@ -34,13 +34,13 @@ func run() error {
 		return errors.Trace(err)
 	}
 
-	f, err := os.Create("tmp/jnet.jsonnet")
+	dir, err := ioutil.TempDir("", "")
```

Comment:
Pásale un nombre que prefije las carpetas:
```suggestion
	dir, err := ioutil.TempDir("", "jnet")
```